mkdocs.yml

```diff
@@ -34,3 +34,5 @@ nav:
 
 theme:
   name: material
+  logo: artwork/logos/png_256/AsahiLinux_logomark.png
+  favicon: artwork/logos/png_256/AsahiLinux_logomark.png
```

Comment:
Picked at random, I dunno if there's better options we should use here

Reply:
Should we pull in the 32x32 logo for the favicon or does mkdocs create a scaled version itself when generating the pages?

Reply:
I'm not sure actually, I've always used pregenerated images of the right size in other deployments